lib/utils/ConfigParser.js

```diff
@@ -272,6 +253,40 @@ class ConfigParser {
         return this._capabilities
     }
 
+    /**
+     * sets config attribute with file paths from filtering
+     * options from cli argument
+     *
+     * @param  {String} filterObj  list of files in a string from
+     * cli argument
+     * @return {String} filterType  either exclude or spec
+     */
+    setFilePathToFilterOptions (filterObj, filterType) {
```

Comment:
function head should be this:

```js
setFilePathToFilterOptions (filterObj, exclude=false) {
```

as I don't see what other filtertype could be used in the future here.